web.test.tests/web.test.tests/Pages/DepositPage.cs

```diff
@@ -0,0 +1,59 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Text;
+using System.Threading;
+
+namespace web.test.tests.Pages
+{
+    class DepositPage
+    {
+        private IWebDriver driver;
+
+        public DepositPage(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+
+        public IWebElement AmountField => driver.FindElement(By.Id("amount"));
+        public IWebElement RateField => driver.FindElement(By.Id("percent"));
+        public IWebElement TermField => driver.FindElement(By.Id("term"));
+        public String AmountValue => AmountField.GetAttribute("value");
+        public String RateValue => RateField.GetAttribute("value");
+        public String TermValue => TermField.GetAttribute("value");
+        public SelectElement SelectStartDay => new SelectElement(driver.FindElement(By.Id("day")));
+        public SelectElement SelectStartMonth => new SelectElement(driver.FindElement(By.Id("month")));
+        public SelectElement SelectStartYear => new SelectElement(driver.FindElement(By.Id("year")));
+        public String StartDay => SelectStartDay.SelectedOption.Text;
+        public String StartMonth => SelectStartMonth.SelectedOption.Text;
+        public String StartYear => SelectStartYear.SelectedOption.Text;
```

Comment:
You can add a property StartDate. e.g. 
```suggestion
        public SelectElement SelectStartDay => new SelectElement(driver.FindElement(By.Id("day")));
        public SelectElement SelectStartMonth => new SelectElement(driver.FindElement(By.Id("month")));
        public SelectElement SelectStartYear => new SelectElement(driver.FindElement(By.Id("year")));


public DateTime StartDate
{
    get
    {
        var startDay => SelectStartDay.SelectedOption.Text;
        var startMonth => SelectStartMonth.SelectedOption.Text;
        var startYear => SelectStartYear.SelectedOption.Text;
        
        return DateTime.Parse($"{startYear} {startMonth} {startDay}");
    }
    set
    {
        SelectStartYear.SelectValue(value.ToString("yyyy"));
        SelectStartMonth.SelectValue(value.ToString("MMMM"));
        SelectStartDay.SelectValue(value.ToString("dd"));
    }
}
```

Reply:
Cannot understand how 'set' should work

Reply:
When you assign some value to the property, then the 'set' method is executed with 'value' parameter.

var date = new DateTime(2021, 11, 25);
depositePage.StartDate = date; // the 'setter' will select year/month/day in dropdowns.

Reply:
Got it. Thanks. Done.